en/ros/ros2_offboard_control.md

```diff
@@ -0,0 +1,189 @@
+# ROS 2 Offboard Control Example
+
+:::warning
+*Offboard* control is dangerous.
+If you are operating on a real vehicle be sure to have a way of gaining back manual control in case something goes wrong.
+:::
+
+:::warning
+ROS 2 interaction with PX4 through the *microRTPS* bridge requires that the user understands how the PX4 internals work!
```

Comment:
do you mean microRTPS - elsewhere I have changed to "PX4-Fast RTPS(DDS) Bridge "

Reply:
No. Please revert to microRTPS, since it't the name which is commonly use. the `PX4-Fast RTPS(DDS) Bridge` should only be in the title of micrortps.md.

Reply:
OK. Then in the micrortps.md I will add a note indicating that it is also known as micrortps bridge.